packages/terra-search-field/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ ChangeLog
 
 Unreleased
 ----------
+### Removed
+* Removed `terra-form-input` from dependencies.
+
+### Changed
+* Updated `terra-form-input` to `native input`. 
```

Comment:
I think this entry should be something similar to "Removed the terra-form-input and replaced it with a native input"